test/saithrift/tests/saiacl.py

```
@@ -63,7 +63,7 @@ def runTest(self):
         try:
             send_packet(self, 2, str(pkt))
             verify_packets(self, exp_pkt, [1])
-        finally:
+        except IPSendRecvErr:
```

catch all exceptions, can clean up the system. after clean up, should return.